### PR TITLE
java_cert: remove useless 'return' statement

### PR DESCRIPTION
**Pull request by pilou-**
##### SUMMARY
java_cert:
- remove useless `return` statement
- check_mode: small refactor
- doc: use `C()` formatting function

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
java_cert

##### ANSIBLE VERSION
```paste below
2.8
```

### CODE REVIEW

**Comment by pilou-** [comment]
cc @Yannig

File: lib/ansible/modules/system/java_cert.py [hunk 1]
```diff
@@ -334,10 +322,18 @@ def main():
 
     if state == 'absent':
```
Comment (webknjaz) [inline lib/ansible/modules/system/java_cert.py]: I'd collapse if blocks to save on nesting +1 to readability:
```suggestion
     if state == 'absent' and cert_present:
```

File: lib/ansible/modules/system/java_cert.py [hunk 2]
```diff
@@ -334,10 +322,18 @@ def main():
...
         if cert_present:
+
+            if module.check_mode:
+                module.exit_json(changed=True)
+
             delete_cert(module, executable, keystore_path, keystore_pass, cert_alias)
 
     elif state == 'present':
```
Comment (webknjaz) [inline lib/ansible/modules/system/java_cert.py]:
```suggestion
     elif state == 'present' and not cert_present:
```

**Comment by Yannig** [comment]
Look good to me.

@pilou- what next with my PR? I wait for this PR to be merged then I rebase mine against this new version?

**Comment by pilou-** [comment]
> what next with my PR? I wait for this PR to be merged then I rebase mine against this new version?

There is no rush to merge this one, once #45784 has been updated and merged, I will rebase this one (another way to do is to add commits from this PR to #45784, then this one could be closed).

**Comment by Yannig** [comment]
> There is no rush to merge this one, once #45784 has been updated and merged, I will rebase this one (another way to do is to add commits from this PR to #45784, then this one could be closed).

I'll pick your commit to my PR.

**Comment by Akasurde** [comment]
superseded by #52879

**Pull request closed without merging**